noxfiles/test_docker_nox.py

```diff
@@ -13,6 +13,7 @@ def test_single_tag(self) -> None:
             "buildx",
             "build",
             "--push",
+            "--provenance=false",
```

Comment:
Setting `provenance` to false to prevent manifest from being exported as a list. This is what allows us to do the docker manifest create, amend, push in the `CreateManifests` workflow step.